test-requirements.txt

```diff
@@ -0,0 +1,5 @@
+coverage==3.7.1
+mock==1.0.1
+nose==1.3.3
+flake8==2.3.0
+objgraph==3.4.0
```

Comment:
👍 thanks for cleaning this up.

However, I personally prefer to call it `requirements-test.txt` so that if viewing a folder, all the `requirements*.txt` files will be next to each other. Otherwise very easy to overlook that there are separate files.

Reply:
Done